pkg/initialise/initialise.go

```diff
@@ -0,0 +1,92 @@
+package initialise
+
+import (
+	"path/filepath"
+
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/api"
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/download/yamlcreator"
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/util"
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/util/files"
+)
+
+//CreateTemplate Creates a blank set of monaco folders and files for each supported API
+func CreateTemplate(workingDir string) error {
+
+	workingDir = filepath.Clean(workingDir)
+	util.Log.Info("Initialising Monaco Demo Folders")
+	projectsFolder := workingDir + "/projects"
```

Comment:
Please use `filepath.Join` here, as this would not work on windows, as they use a different 
path separator `\`. 

(It would be best to replace all string concatenations with `filepath.Join`)